main.cpp

```diff
@@ -172,38 +195,54 @@ struct ButtonFactory
         std::string getButtonColor( std::string sku );
     };
 };
-/*
-Thing 2) Alien detection agency
-5 properties:
-    1) number of aliens on ground
-    2) fleet size
-    3) intelligence (IQ)
-    4) global coordinates detected from
-    5) case number
-3 things it can do:
-    1) process reports by field agents
-    2) filter out false reports
-    3) publish vetted reports
- */
+
+
+void ButtonFactory::makeButton(std::string sku, int amount)
+{
+    Button button;
+    button.skuInfo = sku;
+    amount += button.stockQuantity;
+}
```

Comment:
This function should return a `Button`
Why are you adding `button.stockQuantity;` to `amount`? 
Even though this project makes you write code for the sake of writing code, it's important to still write code that **makes sense.**
- [ ] revise